ragna/deploy/_ui/app.py

```diff
@@ -23,32 +23,45 @@
 pn.config.browser_info = True
 
 
-HERE = Path(__file__).parent
-# CSS = HERE / "css"
-IMGS = HERE / "imgs"
-RES = HERE / "resources"
-
-
 class App(param.Parameterized):
     def __init__(self, *, url, api_url, origins):
         super().__init__()
-        ui.apply_design_modifiers()
+
+        # Apply the design modifiers to the panel components
+        # It returns all the CSS files of the modifiers
+        self.css_filepaths = ui.apply_design_modifiers()
         self.url = url
         self.api_url = api_url
         self.origins = origins
 
     def get_template(self):
+        # A bit hacky, but works.
+        # we need to preload the css files to avoid a flash of unstyled content, especially when switching between chats.
```

Comment:
Should we report this to panel as well? Seems like a useful thing to have in general.

Reply:
Yes. I'd say the feature request is "add a placeholder for custom HTML inside the <head>"

Reply:
Did we do that yet?

Reply:
@pierrotsmnrd will do it afterwards.